java/client/src/test/java/glide/ExceptionHandlingTests.java

```diff
@@ -84,7 +83,7 @@ public void channel_is_closed_when_disconnected_on_command() {
         var channelHandler = new TestChannelHandler(callbackDispatcher);
         var commandManager = new CommandManager(channelHandler);
 
-        var future = commandManager.submitNewCommand(createDummyCommand(), r -> null);
+        var future = commandManager.submitNewCommand(CustomCommand, new String[0], r -> null);
         callbackDispatcher.completeRequest(null);
         var exception = assertThrows(ExecutionException.class, future::get);
         // a ClosingException thrown from CallbackDispatcher::completeRequest and then
```

Comment:
custom command with empty args list shouldn't close the client (e.g. raise ClosingException)